locale/misc/fields/en.yaml

```diff
@@ -10,3 +10,9 @@ last_name: Last name
 phone: Phone number
 street_address: Street address
 zip: Zip code
+genderOptions:
+  f: Female
+  m: Male
+  o: Other
+  null: Unknown
+  _: Unknown
```

Comment:
There should never be a case where the value (from the API) is `_`. The underscore is only used in some UI implementations because it's not possible to store `null` as a value in HTML attributes.

If there is a case where `_` occurs in the data from the API, that's a bug. Have you seen that happen?